docs/QUICK_START.md

```diff
@@ -0,0 +1,193 @@
+# mLRS Documentation: Quick Start Guide #
+
+([back to main page](../README.md))
+
+This page provides hints, suggestions and links to help get new users started with mLRS quickly.
+
+This guide assumes you want to use mLRS in it's core use case to provide a single radio link to carry both RC control and full bi-directional MAVLink Telemetry.  When we discuss configuration of your flight controller, we assume you are running ArduPilot.  We also assume you use OpenTX or EdgeTX and that you will connect a ground station and/or use the [Yaapu Telemetry App](CRSF.md).
```

Comment:
Should be 'its', 'it's' is a contraction of 'it is'

Reply:
I know that, but, fingers don't.

Reply:
:+1: